docs/user-guide/configure-certificates.md

```diff
@@ -10,21 +10,61 @@ Zowe provides the ability to generate a certificate using the `zwe init certific
 
+
 ## Keystore versus key ring
 
-Zowe supports certificates that are stored either in a USS directory **Java KeyStore** `.p12` format or else held in a **z/OS Keyring**.  z/OS keystore are the preferred choice for storing certificates where system programmers are already familiar with their operation and usage.  The user ID setting up a keystore and connecting it with certificates requires elevated permissions, and in scenarios where you need to create a Zowe sandbox environment or for testing purposes and your TSO user ID doesn't have authority to manipulate key rings, USS keystores are a good alternative.  
+Zowe supports certificates that are stored either in a USS directory **Java KeyStore** `.p12` format or else held in a **z/OS Keyring**. z/OS keystore are the preferred choice for storing certificates where system programmers are already familiar with their operation and usage. The user ID setting up a keystore and connecting it with certificates requires elevated permissions, and in scenarios where you need to create a Zowe sandbox environment or for testing purposes and your TSO user ID doesn't have authority to manipulate key rings, USS keystores are a good alternative.
+
+### PKCS12 certificates in a keystore
+
+Zowe is able to use PKCS12 certificates that are stored in USS. This certificate is used for encrypting TLS communication between Zowe clients and the Zowe z/OS servers, as well as intra z/OS Zowe server to Zowe server.  Zowe uses a `keystore` directory to contain its external certificate, and a `truststore` directory to hold the public keys of servers it communicate with (for example z/OSMF).  
+
+Using USS PKCS12 certificates is useful for proof of concept projects using a self signed certificates.  For production usage of Zowe it is recomended to work with certificates held in z/OS keyrings.  Working with z/OS keyrings may require system administrator priviledges and working with your z/OS security team, so the self signed PKCS12 path is provided to assist with configuring and launching test and scratch Zowe instances.
+
+### JCERACFKS certificates in a key ring
+
+Zowe is able to work with certificates held in a **z/OS Keyring**.  
+
+The JCL member `.SZWESAMP(ZWEKRING)` contains the security commands to create a keyring named `ZoweKeyring` and manage the certificate and certificate authoritie (CA) used by Zowe's servers to encrypt TLS communications.  The JCL contains commands for three z/OS security managers: RACF, TopSecret, and ACF/2.
+
+There are two ways to configure and submit `ZWEKRING`.
+
+- Customize and submit the `ZWEKRING` JCL member.
+- Customize the `zowe.setup.certificate` section in `zowe.yaml` and use the `zwe init certificate` command. 
+
+If you use the `zwe init certificate` command this will prepare a customized JCL member using `ZWEKRING` as a template.  
+
+A number of keytring scenarios are supported
```

Comment:
I think you need a colon: "A number of keytring scenarios are supported:"

Also, I think there's a typo in "keytring"